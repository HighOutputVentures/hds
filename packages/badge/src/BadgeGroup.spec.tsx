describe("BadgeGroup", () => {
  it.todo("Should render badge group");
});
