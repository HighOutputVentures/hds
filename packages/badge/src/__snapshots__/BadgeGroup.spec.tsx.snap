// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BadgeGroup Should match BadgeGroup snapshot 1`] = `
Array [
  <div
    className="css-fac20c"
    data-testid="hds.badge-group.container"
    role="alert"
  >
    <div
      className="css-d5rpjh"
      data-testid="hds.badge-group.title"
    >
      Title
    </div>
    <div
      className="css-54jk18"
      data-testid="hds.badge-group.description"
    >
      Description
      <button
        className="css-0"
        role="button"
      >
        <svg
          className="chakra-icon css-1pyyh8m"
          fill="none"
          focusable={false}
          viewBox="0 0 12 12"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9 3L3 9M3 3L9 9"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="1.5"
          />
        </svg>
      </button>
    </div>
  </div>,
  <span />,
]
`;
